apps/portal/start-dev.sh

```
@@ -33,64 +33,136 @@ YELLOW='\033[1;33m'
+  ARGOCD_PASS=$(kubectl -n argocd get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | decode_base64)
+
+  echo -e "${YELLOW}🚀  Starting ArgoCD Port-Forward (localhost:8080)...${NC}"
+  kubectl port-forward -n argocd svc/argocd-server 8080:443 > /dev/null 2>&1 &
+  PIDS+=($!)
+  sleep 2
+
+  TOKEN_JSON=$(curl -k -s -X POST -H "Content-Type: application/json" \
+    -d "{\"username\":\"admin\",\"password\":\"$ARGOCD_PASS\"}" \
+    https://127.0.0.1:8080/api/v1/session)
+
+  ARGOCD_AUTH_TOKEN=$(echo "$TOKEN_JSON" | sed 's/.*"token":"\([^"]*\)".*/\1/')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)